src-v2/NTS.Domain.Setup/Entities/Competition.cs

```diff
@@ -14,7 +14,7 @@ public class Competition : DomainEntity, ISummarizable, IImportable
     private List<Contestant> _contestants = new();
 
     [JsonConstructor]
-    public Competition(string name, CompetitionType type, DateTimeOffset start)
+    private Competition(string name, CompetitionType type, DateTimeOffset start)
     {
         this.Name = name;
```

Comment:
Validate:
- CompetitionType is not default value (0)
- DateTimeOffset is not in the past